receiver/mongodbatlasreceiver/documentation.md

```diff
@@ -0,0 +1,130 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# mongoatlasreceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
```

Comment:
If you view this md the table isn't formed and I wonder if the attribute list is throwing it...

Reply:
I think it is, but since it's autogenerated I don't think I've got a lot of control over this formatting.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6297